pennylane/pulse/transmon.py

```diff
@@ -88,7 +88,7 @@ def transmon_interaction(
         d (int): Local Hilbert space dimension. Defaults to ``d=2`` and is currently the only supported value.
```

Comment:
`transmon_drive` is missing from the see-also box in line 75

Reply:
✅